packages/runtime/container-runtime/src/summary/documentSchema.ts

```diff
@@ -219,6 +224,7 @@ const documentSchemaSupportedConfigs = {
 	explicitSchemaControl: new TrueOrUndefinedMax(), // once new behavior shows up, it's sticky
 	idCompressorMode: new IdCompressorProperty(["delayed", "on"]),
 	opGroupingEnabled: new TrueOrUndefined(),
+	gcOpSupported: new TrueOrUndefined(),
```

Comment:
Why is it `gcOpSupported`, when all the other places use `gcOp`?
If that's the bug, what is missing in terms of coverage to diagnose this bug? I'd think that existing UTs should fail

Reply:
Oops yep something should fail

Reply:
Some unit tests failed. I have a concept of how we can get compile-time protection (or remove redundant declaration of type and value), I'll try it in another PR if I have time